packages/permission_handler/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## NEXT
```

Comment:
When should we use `NEXT` in the change log?

Reply:
The convention has been taken from:
https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog

It's used when the change doesn't involve a version update but still needs to be documented.

Reply:
Oh, thank you for your answer.
According to the guide, it seems like every PR should include a change log as much as possible(however it doesn't seem to write a change log enough even in upstream.). what do you think about this?

Reply:
> This policy exists both to make it easier for maintainers to see a record of all changes to a package, and because some changes (e.g., updates to examples) that do not need to be published may still be of interest to clients of a package.

It's a good practice to record ALL changes but not every member cares about it.😢 At least I'm trying my best.